t/25-cache.t

```diff
@@ -0,0 +1,247 @@
+#! /usr/bin/perl
+
+# Copyright (c) 2016 SUSE LINUX GmbH, Nuernberg, Germany.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+BEGIN {
+    unshift @INC, 'lib';
+    $ENV{OPENQA_TEST_IPC} = 1;
+}
+
+use strict;
+use warnings;
+
+use Test::More;
+use Test::Warnings;
+use Test::MockModule;
+use OpenQA::Utils;
+use OpenQA::Worker::Cache;
+use DBI;
+use File::Path qw(remove_tree make_path);
+use File::Spec::Functions qw(catdir catfile);
+use Digest::MD5 qw(md5);
+use Cwd qw(abs_path getcwd);
+
+use Mojo::IOLoop;
+use Mojolicious;
+use Mojo::Server::Daemon;
+use Mojolicious::Lite;
+
+
+
+# create Test DBus bus and service for fake WebSockets call
+# my $ws = OpenQA::WebSockets->new;
```

Comment:
?

Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit